iina/KeyBindingTranslator.swift

```diff
@@ -114,12 +119,16 @@ class KeyBindingTranslator {
   static func string(fromCriterions criterions: [Criterion]) -> String {
     var mapped = criterions.filter { !$0.isPlaceholder }.map { $0.mpvCommandValue }
 
-    let firstName = (criterions[0] as! TextCriterion).name
+    let firstCriterion = criterions[0] as! TextCriterion
```

Comment:
If this is so, then an `assert` at the top of the function would give a better error message than "Index out of range".